azurerm/internal/services/web/tests/helpers_test.go

```diff
@@ -0,0 +1,9 @@
+package tests
+
+import (
+	"os"
+)
+
+func skipStaticSite() bool {
+	return os.Getenv("ARM_TEST_GITHUB_TOKEN") == ""
```

Comment:
This should probably have a `TF` prefix, rather than `ARM` as these are for configuring the client itself, rather than test flags.
```suggestion
	return os.Getenv("TF_ACCTEST_GITHUB_TOKEN") == ""
```